### DISCUSSION TITLE
Create a LinkFree editor

### DISCUSSION THREAD

**Discussion by Pierstoval** [EddieHubCommunity/LinkFree#1783] [Ideas]
Hello!

I'm of the ones that would encourage people to self-host open-source apps for their needs instead of relying on big corps doing it.

Today I recommend LinkFree over linktr.ee, and the "Single user mode" is perfect for that!

However, it lacks one single thing: a simple CRUD backend to edit one's links.

I was wondering whether there were decisions made over time in the team to include such backend as part of the app, or if there were external initiatives in favor of this? It's so that we could push a static site to prod, but also have a backend to edit their links.

Not talking about analytics or stuff, just editing content in a more fashionable way than editing a JSON file :)

**Comment by eddiejaoude** [comment]
We now have forms for CRUD, and also analytics. Have a look and let us know what you think
